app/assets/js/hooks/use-wrap-action-with-verify-dialog.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import useIntroduction from './use-introduction';
+
+export default function useWrapActionWithVerifyDialog( {
```

Comment:
```suggestion
export default function useWrapActionWithConfirmationDialog( {
```

hmm, IDK
feels too long, and I'm not sure the name is good enough, maybe let's talk about it

Reply:
Changed, but lets talk

Reply:
Done